src/components/containers/seo.js

```diff
@@ -186,25 +205,41 @@ const SEO = ({ description, meta, title, no_index, has_organization_schema, meta
             {has_organization_schema && (
                 <script type="application/ld+json">{JSON.stringify(organization_schema)}</script>
             )}
-
-            {!is_non_localized &&
-                languages.map((locale) => {
-                    if (!(locale === 'ach')) {
-                        const replaced_local = locale.replace('_', '-')
-                        const is_default = locale === 'en' || locale === 'x-default'
-                        const href_lang = is_default ? '' : `/${replaced_local}`
-                        const href = `${site_url}${href_lang}${current_page}`
-
-                        return (
-                            <link
-                                rel="alternate"
-                                hrefLang={replaced_local}
-                                href={href}
-                                key={replaced_local}
-                            />
-                        )
-                    }
-                })}
+            {is_eu
```

Comment:
@meenakshi-deriv  As the only difference between these two maps is just the array that is rendered, So I think this can be merged to avoid duplicate code. However, it's just a suggestion.